dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -1334,6 +1408,9 @@ CFileDefExt::CountFolderAndFiles(HWND hwndDlg, LPWSTR pwszBuf, UINT cchBufMax, D
 
     do
     {
+        ZeroMemory(WorkBuffer,sizeof(WorkBuffer));
+        wcscpy(WorkBuffer,PathBuffer);
+        wcscat(WorkBuffer,wfd.cFileName);
```

Comment:
Not using StringCchCopyW / StringCchCatW instead?

Reply:
OK